scss/_patterns_card.scss

```diff
@@ -0,0 +1,29 @@
+// Card patterns
+@mixin vf-p-card {
+
+  .p-card {
+    background: $color-light;
+    border-radius: 2px;
+    border: 1px solid $color-mid-grey;
+    padding: 1rem;
+    text-align: center;
```

Comment:
The pattern design visual has all text centered - if you feel that's incorrect and by default all text should be left aligned, using the util for exceptions then the design should be amended - check with @yaili